rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/integration/ComponentTest.java

```diff
@@ -143,8 +150,10 @@ public void should_update_component_valid() throws IOException, TException {
         Map<String, String> body = new HashMap<>();
         body.put("name", updatedComponentName);
         body.put("invalid_property", "abcde123");
+        String url = "http://localhost:" + port + "/api/components/" + componentId;
+        UriComponentsBuilder builder = UriComponentsBuilder.fromUriString(url).queryParam("allowDuplicateAttachment", "true");
```

Comment:
Maybe add another test, where duplicates are not allowed and you try to update with a duplicate.